numpy/core/src/private/ufunc_override.c

```diff
@@ -8,148 +8,172 @@
 
 #include "ufunc_override.h"
 
-static void
+
+static int
 normalize___call___args(PyUFuncObject *ufunc, PyObject *args,
-                    PyObject **normal_args, PyObject **normal_kwds,
-                    int nin)
+                        PyObject **normal_args, PyObject **normal_kwds,
+                        int nin, int nout)
```

Comment:
I'm going to add a big warning note for now: this is outside of the scope of this should-have-been-small fix -- but I'll also mention it as a possible issue in the main thread.